systest/exclude/tempest_12.1.2_overcloud_vxlan.yaml

```diff
@@ -0,0 +1,7 @@
+{
+    "exclude": ["test_health_monitor_basic",
```

Comment:
tempest_12.1.2_overcloud_vxlan target doesn't exist anymore. It is renamed to tempest_12.1.2_overcloud so we can skip merging this file forward, also it can be removed from liberty or renamed to tempest_12.1.2_overcloud.yaml. Since we don't have this target in the liberty nightly anymore we didn't run into problems.